sway-core/src/semantic_analysis/ast_node/declaration/abi.rs

```diff
@@ -73,6 +73,10 @@ impl ty::TyAbiDeclaration {
                         ctx.decl_engine.insert(method),
                     ));
                 }
+                TraitItem::Constant(_) => {
+                    // not implemented for now since abis and traits will be unified soon
```

Comment:
I think @anton-trunov said we may not be doing that for a while. Is it difficult to fill this in in a follow up PR?